pkgs/applications/editors/vscode/default.nix

```diff
@@ -1,9 +1,9 @@
 { stdenv, lib, fetchurl, unzip, atomEnv, makeDesktopItem,
-  gtk2, wrapGAppsHook, libXScrnSaver, libxkbfile, libsecret }:
+  gtk2, wrapGAppsHook, libXScrnSaver, libxkbfile, libsecret,
+  isInsiders ? false }:
```

Comment:
How does one actually set this? I was hoping there would be `vscode` and `vscode-insiders` packages, but no suck luck.

Reply:
Here's my overlay, not really deterministic but gets me my nightly releases.
```nix
vscode = (super.vscode.override {
    isInsiders = true;
  }).overrideAttrs(_: rec {
    name = "${pname}-${version}";
    pname = "vscode-insiders";
    version = "nightly";

    src = builtins.fetchurl {
      name = "VSCode_latest_linux-x64.tar.gz";
      url = "https://vscode-update.azurewebsites.net/latest/linux-x64/insider";
    };
  });
```

Reply:
Ah, I see, I'll do the same, thanks!